src/styles/blocks/header.scss

```diff
@@ -0,0 +1,74 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  background-color: #fff;
+}
+
+.logo {
+  height: 40px;
+  width: 40px;
+  display: block;
+}
+
+.nav {
+  &-list {
+    display: flex;
+    padding: 0;
+    margin: 0;
+    list-style: none;
+  }
+
+  &-item {
+    margin-left: 20px;
+
+    &:first-child {
+      color: $main-color;
+      margin-left: 0;
+    }
+  }
+
+  &-link {
+    position: relative;
+
+    display: flex;
+    align-items: center;
+    height: 60px;
+
+    text-decoration: none;
+    text-transform: uppercase;
+    font-weight: 500;
+    white-space: nowrap;
+    font-size: 12px;
+    color: $nav-links-color;
+    transition: 300ms;
+
+    cursor: pointer;
+
+    &:hover {
+      color: $main-color;
+      color: $nav-links-hover-color;
+    }
+
+    &.is-active {
```

Comment:
apply .is-active class correctly in your html

Reply:
My BEM was ruined when I used 'is-active' class in the nav-link as the second block:
error Tag can't have more [nav-link] (one-block) than one block
error Tag can't have more [is-active] (one-block) than one block.
Because of that test was ruined also, it was searching for the class 'is-active', but I couldn`t add it, because of linting problems. What do I need to do?